Dinar1/Rechner.java

```diff
@@ -0,0 +1,56 @@
+package ru.codecrafts;
+import java.util.ArrayList;
+import java.util.Scanner;
+public class Rechner
+{
+    public static void main(String[] args)
+    {
+            ArrayList<Integer> digits = new ArrayList<>();
+            Scanner big = new Scanner(System.in);
+            String button = "yes";

```

Comment:
You don't need this variable

Reply:
Why don't I need it? Later I use it in the cycle for.